chapter_12_organising_test_files.asciidoc

```diff
@@ -547,6 +552,8 @@ Like this:
 ((("Python 3", "lambda functions")))
 `lambda` in Python is the syntax for making a one-line, throwaway function--it
 saves you from having to use `def..():` and an indented block:
```

Comment:
Apparently I can't make my comment below a committable change, so I'll just suggest it again.
I tried the `keep-together` but it didn't work within a code block. How about cheating by adding a word:
```
saves you from having to use the regular `def..():` and an indented block:
```